proj/src/main/webapp/style.css

```diff
@@ -18,48 +17,191 @@ html{
 .hidden{
     visibility:hidden;
 }
+
 /* main: welcome page */
 /* view: mobile */
+
+/* NOTE: remove later, using this for organization */
```

Comment:
By "later", did you mean now?

Reply:
I was thinking about leaving it until all the visual styling/organization is done

Reply:
Okay.

Reply:
btw, I think in general you should use `TODO:` instead of `NOTE:` to mark things that need to be done in the future just for consistency